pkg/lint/lintersdb/manager.go

```diff
@@ -715,6 +715,13 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
 			WithPresets(linter.PresetStyle).
 			WithURL("https://github.com/yeya24/promlinter"),
 
+		linter.NewConfig(golinters.NewProtoGetter()).
+			WithSince("v1.55.0").
+			WithPresets(linter.PresetBugs).
+			WithLoadForGoAnalysis().
+			WithAutoFix().
```

Comment:
brew updated my golangci-lint and I suddenly started getting all my proto field accesses rewritten. This is not cool, they are not bugs and I didn't ask for this "fix". I suggest this behavior should not be the default.

Reply:
This linter is not enabled by default, you should have turned it on yourself.
If you have all linters enabled, you can disable `protogetter` through the configuration by adding a linter to the `disable` array.
https://golangci-lint.run/usage/configuration/#linters-configuration

Reply:
@ash2k hi!

`WithAutoFix()` affects documentation only.
Do you sure that you run `golangci-lint` without `--fix` flag?

Reply:
Looks like I have `fix: true` in my config. I used [the reference config](https://github.com/golangci/golangci-lint/blob/5a0f059f450ce822d2e0ad3167331f5bcaa9559b/.golangci.reference.yml#L2594) (a long time ago) as a template and this behavior was and still is the default. I didn't know fixing was enabled as no other linter ever made any fixes. In fact, I didn't know linters can autofix issues they find.

I've disabled the linter and autofixing for now.

> If you have all linters enabled

Yes, I do. I don't have time to inspect each version and learn about the new linters and their configuration so this is convenient because that's how I discover there is a new linter in a new version and that it found something suspicious. I just didn't expect it to mess with me code :)

Thank you for educating me. I've corrected my config.

Reply:
@ash2k Also it may not be obvious, but this config describes default values in field comments, and uses an example of non-default values in the fields themselves 🙂 

Example:
```yml
   # Independently of option `exclude` we use default exclude patterns, it can be disabled by this option.
   # Default: true.
   exclude-use-default: false
```

The default `fix` option value is `false` but comment was missed 😢  We'll fix it